pkg/tekton/pipelines.go

```diff
@@ -273,13 +294,17 @@ func CreateOrUpdatePipeline(tektonClient tektonclient.Interface, ns string, crea
 }
 
 // PipelineResourceName returns the pipeline resource name for the given git repository, branch and context
-func PipelineResourceName(gitInfo *gits.GitRepository, branch string, context string) string {
+func PipelineResourceName(gitInfo *gits.GitRepository, branch string, context string, pipelineType PipelineType) string {
```

Comment:
@abayer hmm, I see quite a few names getting truncated, even before the addition of the pipeline type. For example, the names for the environment build. Is this not really an issue?

Reply:
Correct. The CRD name is just cosmetic, really.

Reply:
I see. I guess the object ID is used internally somehow. Still it gets a bit odd looking at things like `kubectl get pods` etc, everything where the name is visible.

This 64 char limit is really a pita imo.